src/pages/AppList.svelte

```diff
@@ -1,4 +1,7 @@
 <script>
+	import { fade } from 'svelte/transition';
```

Comment:
I appreciate the enthusiasm and desire to experiment, but let's leave the animated transitions to a seperate PR / issue. :) This helps describe why: https://homes.cs.washington.edu/~mernst/advice/version-control.html#logical-unit

For now I'd like to mostly concentrate on making sure that all the required UI elements are there, rather than on how they look.

Reply:
I thought it would make the boring UI looks nicer :sweat_smile: 
Will be careful about it in future.